tests/aws/services/cloudwatch/test_cloudwatch_metrics.py

```diff
@@ -123,3 +123,37 @@ def _wait_for_lambda_metric(
         )
         assert res["MetricDataResults"][0]["Values"] == expected_return
         return res
+
+
+class TestSqsApproximateMetrics:
+    @markers.aws.validated
+    def test_sqs_approximate_metrics(self, aws_client, sqs_create_queue):
+        queue_names = []
+        for _ in range(0, 10):
+            q_name = f"my-test-queue-{short_uid()}"
+            sqs_create_queue(QueueName=q_name)
+            queue_names.append(q_name)
+
+        for queue in queue_names:
+            retry(
+                lambda: self._assert_approximate_metrics_for_queue(
+                    aws_client.cloudwatch, queue_name=queue
+                ),
+                retries=70,  # should be reported every 60 seconds on LS
+                sleep=10 if is_aws() else 1,
+            )
+
+    def _assert_approximate_metrics_for_queue(
+        self,
+        cloudwatch_client: "CloudWatchClient",
+        queue_name: str,
+    ):
```

Comment:
stupid question, I have never seen this before: what does the "CloudWatchClient" type do? I couldn't really find anything useful online, and when I enter it like that, pycharm tells me that the type could be Any.

Reply:
It's a type hint, meaning that the `cloudwatch_client` should be of type `CloudWatchClient`. 

We have a conditional import for the type:
```
if TYPE_CHECKING:
    from mypy_boto3_cloudwatch import CloudWatchClient
```
[Here are some docs](https://docs.python.org/3.10/library/typing.html#typing.TYPE_CHECKING).